8Knot/db_manager/augur_manager.py

```diff
@@ -255,47 +269,38 @@ def is_org(self, org):
+            self.multiselect_startup()
+
         return org in self.org_names
 
     def initial_multiselect_option(self):
         """Getter method for the initial multiselect option.
-            May be overwritten by the environment.
+            Provides a lightweight default without triggering database queries at startup.
 
         Returns:
             dict(value, label): first thing the multiselect will represent on startup
         """
         try:
-            if self.initial_search_option is None:
-                # default the initial multiselect option to the
-                # first item in the list of options.
-
-                self.initial_search_option = self.multiselect_options[0]
-
-                if os.getenv("DEFAULT_SEARCHBAR_LABEL"):
-                    logging.warning("INITIAL SEARCHBAR OPTION: DEFAULT OVERWRITTEN")
-                    default_label = os.getenv("DEFAULT_SEARCHBAR_LABEL")
-
-                    # search through available options for the specified overwriting default.
-                    found_option = False
-                    for opt in self.multiselect_options:
-                        if default_label == opt["label"]:
-                            # Create a copy of the option with the "repo:" prefix
-                            self.initial_search_option = opt.copy()
-                            # Add "repo:" prefix if it's a repo (integer value)
-                            if isinstance(opt["value"], int):
-                                self.initial_search_option["label"] = f"repo: {opt['label']}"
-                            else:
-                                self.initial_search_option["label"] = f"org: {opt['label']}"
-
-                            logging.warning(f"INITIAL SEARCHBAR OPTION: NEW DEFAULT: {self.initial_search_option}")
-                            found_option = True
-                            break
-
-            return self.initial_search_option
+            if os.getenv("DEFAULT_SEARCHBAR_LABEL"):
+                default_label = os.getenv("DEFAULT_SEARCHBAR_LABEL")
+                logging.warning(
+                    f"INITIAL SEARCHBAR OPTION: Using environment default '{default_label}' without DB lookup"
+                )
+                # Return the environment-specified option directly (assumes it's an org name like "chaoss")
+                return {"label": f"org: {default_label}", "value": default_label.lower()}
+
+            # If no environment default, return a generic first option without loading database
+            # This will be properly resolved when user actually searches or selects
+            return {"label": "org: chaoss", "value": "chaoss"}
+
         except Exception as e:
             logging.error(f"Error in initial_multiselect_option: {str(e)}")
             # Return a fallback option in case of any error
-            return {"label": "repo: Fallback Option", "value": -1}
+            return {"label": "org: chaoss", "value": "chaoss"}
```

Comment:
This should probably be a generic option as there could be augur instances without the chaoss org. i think that was the reason for the -1 value

Reply:
AFAIK the default search option is an environment variable/is configurable so i think this is like a second fallback option and we should probably leave it as is